scripts/contribute/extensions/user-extension.js

```diff
@@ -52,7 +52,7 @@ module.exports = toolbox => {
           toolbox.print.info('Adding new contributor');
 
           Object.assign(user, {
-            github: `https://github.com/${gitHubId}`,
+            github: gitHubId,
```

Comment:
Why is this being removed? Does github understand what gitHubId is in this case?

Reply:
This code stores the data used to generate markdown frontmatter. The component then embeds the ID (from the frontmatter) within the full GitHub url. In this case, it was being done twice (would evaluate to https://github.com/https://github.com/${gitHubId}) –– this is a fix to that problem